sensor_msgs/Image request_image
---
sensor_msgs/Image response_image